sonar-testing-harness/pom.xml

```diff
@@ -23,6 +23,11 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-core</artifactId>
+      <version>1.3</version>
```

Comment:
We usually put the version number into the parent. Is defining the version number here on purpose?

Reply:
good point. Will be fixed.